packages/backend/src/main.ts

```diff
@@ -2,13 +2,21 @@ import { NestFactory } from '@nestjs/core';
 import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 require('dotenv').config();
-import { Logger } from '@nestjs/common';
+import { Logger,ValidationPipe } from '@nestjs/common';

```

Comment:
importy chyba powinny znajdować się nad require().config() ?

Reply:
w tym wypadku nie ma znaczenia, że import jest pod reqire, dlatego że to require('dotenv').config(); jest starszym zapisem importa

Reply:
obalone